index.js

```diff
@@ -68,7 +70,35 @@ function toggleColor(element) {
   }
 }
 
+element.addEventListener("click",function(event) {
+  toggleColor(element)
+})
+
+
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector(".like-button")
+
+likeButton.addEventListener("click", function(event){
+  const likes = document.querySelector('.likes')
+  const numberOfLikes = parseInt(likes.textContent)++
+  likes.textContent = `${numberOfLikes} Likes`
```

Comment:
You've got the right idea here, but the `++` operator won't work this way (you probably saw an error about that). You'd need to write it like this:

```js
  const likes = document.querySelector('.likes')
  let numberOfLikes = parseInt(likes.textContent)
  numberOfLikes++
  likes.textContent = `${numberOfLikes} Likes`
```

or

```js
  const likes = document.querySelector('.likes')
  const numberOfLikes = parseInt(likes.textContent) + 1
  likes.textContent = `${numberOfLikes} Likes`
```